components/expenses/ExpenseStatusTag.js

```diff
@@ -36,6 +36,7 @@ const ExpenseStatusTag = ({ status, ...props }) => {
       letterSpacing="0.8px"
       textTransform="uppercase"
       {...props}
+      data-cy="expense-status-msg"
```

Comment:
It's a detail, but we should not prevent the parent from changing the `data-cy` if it wants to

```suggestion
      data-cy="expense-status-msg"
      {...props}
```